src/coreclr/src/vm/i386/virtualcallstubcpu.hpp

```diff
@@ -329,6 +331,7 @@ struct ResolveStub
     DISPL _backpatcherDispl;        // xx xx xx xx          backpatcherWorker  == BackPatchWorkerAsmStub
     BYTE  part11 [1];               // eb           jmp
     BYTE toResolveStub;             // xx                   resolveStub, i.e. go back to _resolveEntryPoint
+    size_t _stackArgumentsSize;     // xx xx xx xx
```

Comment:
Should all this be under `#ifndef UNIX_X86_ABI` ?

Reply:
You are right, I've forgotten that on x86 Linux we use caller side stack arguments cleanup.